index.html

```diff
@@ -0,0 +1,44 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <link rel="stylesheet" href="styles.css">
+    <title>Dire's Portfolio</title>
+    <script src="https://kit.fontawesome.com/b7a7cffe76.js" crossorigin="anonymous"></script>
+    <link rel="preconnect" href="https://fonts.googleapis.com">
+    <link rel="preconnect" href="https://fonts.gstatic.com" crossorigin>
+  <link href="https://fonts.googleapis.com/css2?family=Poppins:wght@100;200;300;600&display=swap" rel="stylesheet" >
+</head>
+<body>
+  <main>
+      <div class="header">
+       <a href="">My Logo</a>
+      <nav><a href=""><i class="fas fa-bars"></i></a></nav>
+      </div> 
```

Comment:
The container for this section is a `header` tag

- [ ] Kindly use a `header` tag in place of `div`